### PR TITLE
Bug 1219075 -  Disable Gaia lockscreen for b2gdroid

### PR DESCRIPTION
**Pull request by samgiles**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b309cdab8ac09c60ba964bd2c942015409c25482

File: distros/b2gdroid/settings.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "screen.timeout": 0,
+  "lockscreen.enabled": false,
```
Comment (fabricedesre) [inline distros/b2gdroid/settings.json]: nit: we should reorder all these settings in alphabetical order.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1b9fa9b2d8d2f6b4f87bd921142a902f3dfd99e8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a0b068b6ef1d082a680e0e70644bd926793d9ad2

**Pull request closed without merging**